plugins/woocommerce-admin/client/products/fills/more-menu-items/about-the-editor-menu-item.tsx

```diff
@@ -14,20 +13,20 @@ import BlockEditorGuide from '~/products/tour/block-editor/block-editor-guide';
 import { usePublishedProductsCount } from '~/products/tour/block-editor/use-published-products-count';
 
 export const AboutTheEditorMenuItem = ( {
-	onClose,
+	onClick = () => null,
+	onCloseGuide,
 }: {
-	onClose: () => void;
+	onClick: () => void;
+	onCloseGuide: () => void;
 } ) => {
 	const [ isGuideOpen, setIsGuideOpen ] = useState( false );
 	const { isNewUser } = usePublishedProductsCount();
 	return (
 		<>
 			<MenuItem
 				onClick={ () => {
-					recordEvent(
-						'block_product_editor_about_the_editor_menu_item_clicked'
```

Comment:
@pmcpinto Worth noting that this was overlapped with the `product_dropdown_option_click` so I removed this event.